include/geode/basic/attribute_manager.hpp

```diff
@@ -127,8 +127,8 @@ namespace geode
         }
 
         template < template < typename > class Attribute, typename T >
```

Comment:
This is no likely to be accepted. Why do you need that?

You can use a workaround using either geode_unused function of [[maybe_unused]] keyword.

Reply:
Hi @BotellaA . Answer in main thread.